agent/app/oswrapper/oswrapper.go

```diff
@@ -11,26 +11,24 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package ec2
+package oswrapper
```

Comment:
Can this package moved to the `agent/utils`, which possibly will be reused in the future?